front/app/modules/commercial/customizable_navbar/admin/components/NavbarItemForm.tsx

```diff
@@ -58,7 +58,10 @@ const NavbarItemForm = ({
 
   return (
     <FormProvider {...methods}>
-      <form onSubmit={methods.handleSubmit(onFormSubmit)}>
+      <form
+        onSubmit={methods.handleSubmit(onFormSubmit)}
```

Comment:
Is the customizable_navbar work meant to be included in this ticket?

Reply:
Yes, I snuck this test here as I didn't want to open a separate PR just for it.